### ISSUE TITLE
Kendo Toolbar responsive async: buttons added to popup by default

### ISSUE THREAD

**Issue by lucasheight** [telerik/kendo-angular#2527]
**Describe the bug**
When rendering a toolbar with [resizable]="true" and buttons are added async.
Rendered buttons appear in the popup even when not required to be in popup.
Resizing the browser window will make buttons appear in correct place.

**To Reproduce**
1. Create a toolbar as per the sample:
```       
        <p>Set toolbar width</p>
        <input
            #range
            style="width: 100%; display: block;"
            type="range"
            value="100"
            min="0"
            max="100"
            step="1"
            (input)="toolbarWidth = range.value"
        />

        <kendo-toolbar [overflow]="true" [style.width.%]="toolbarWidth">
          <kendo-toolbar-button *ngFor="let item of buttons$ |async" [text]="item.text">
          </kendo-toolbar-button>

        </kendo-toolbar>
        <button (click)="onAddNewButton()">add new button</button>
```
2. Add code to make observable array of items.
```
export class AppComponent {

   private buttonCount:number = 0;
    private _buttons$ = new BehaviorSubject<any[]>([]);
    public buttons$:Observable<any> = this._buttons$.asObservable();

  public onAddNewButton = ()=>{
    let existing:any[]=this._buttons$.getValue();
    this.buttonCount+=1;
    let newButton:string = `Button ${this.buttonCount.toString()}`;
    existing.push({text:newButton});
    this._buttons$.next(existing);
    console.log(`new Button (${newButton}) added.`)
  }

}
```
Stackblitz example [Click Here](https://stackblitz.com/edit/angular-wcbek7)

**Expected behavior**
Expected items to be added in the toolbar and not the popup.

**Screenshots**

**Browser**
 - OS: Win 10
 - Browser : Chrome IE11, FF
 - Version [for example, 22]

**Additional context**

**Comment by michal-ayash** [comment]
I encountered the same issue for custom items with ng-content.
Any plan to fix it?

**Comment by PetarMetodiev** [comment]
Thanks for reporting that. I tried reproducing the scenario with `<ng-content>` but with no success. Here is my [StackBlitz](https://stackblitz.com/edit/angular-zhrruh?file=app/app.component.ts). Please try updating it to described behaviour or provide a new StackBlitz which reproduces the problem.

Having said that, both cases seem to be related since tools are added in an async manner. We will be looking through it.

**Comment by PetarMetodiev** [comment]
Fixed in `kendo-angular-toolbar` v3.1.4
